### ISSUE TITLE
Adding Entity-Relationship Diagram Symbols and Notation

### ISSUE THREAD

**Issue by ChobinhoNL** [excalidraw/excalidraw#8925]
It would be immensly usefull for data modelling. I'm pretty sure it's not too much work, so if anyone can do this (my knowledge doesn't extend to js), I'd be thankfull! <3

![Image](https://github.com/user-attachments/assets/77a5ceef-3280-423a-89f2-f6a9b68f0934)

**Comment by georgekenessey** [comment]
Yes please add this. We would love to get rid of Lucidchart and this is the only thing preventing us from doing so.

**Comment by Mrazator** [comment]
Slight update, there were few more UML arrow heads added. We might consider even more in the future (though there is a plenty).

https://x.com/excalidraw/status/1876276078177816692

**Comment by inc0nnu** [comment]
Hi, I missed this enhancement request earlier and created another issue for a very similar request: [https://github.com/excalidraw/excalidraw/issues/9612](url).

So definitely a +1 from me!

Indeed, quite a few arrow heads are added, which is great, however, for data modelling / ERD's there are definitely 3 missing:
- one and only one
- one or zero
- zero, one or many.

KR, Ivo
